src/hotspot/share/memory/metaspaceShared.cpp

```diff
@@ -612,7 +612,8 @@ void MetaspaceShared::prepare_for_dumping() {
 
 // Preload classes from a list, populate the shared spaces and dump to a
 // file.
-void MetaspaceShared::preload_and_dump(TRAPS) {
+void MetaspaceShared::preload_and_dump() {
```

Comment:
I would have kept the Thread argument to avoid the implicit Thread::current() in EXCEPTION_MARK. But it is a margin call either way.